File: goldeneye/src/main/java/co/infinum/goldeneye/SimpleTextureListener.java
package co.infinum.goldeneye;

import android.graphics.SurfaceTexture;
import android.view.TextureView;

public abstract class SimpleTextureListener implements TextureView.SurfaceTextureListener {

    @Override
    public boolean onSurfaceTextureDestroyed(SurfaceTexture surface) {
        return false;
    }

    @Override
    public void onSurfaceTextureUpdated(SurfaceTexture surface) {
    }
}

File: goldeneye/src/main/java/co/infinum/goldeneye/SimpleTextureListener.kt
package co.infinum.goldeneye

import android.graphics.SurfaceTexture
import android.view.TextureView

internal abstract class SimpleTextureListener : TextureView.SurfaceTextureListener {
    override fun onSurfaceTextureUpdated(surface: SurfaceTexture?) {
    }

    override fun onSurfaceTextureDestroyed(surface: SurfaceTexture?) = true
}